File: C/ 2717 Tempo do Duende.c
#include <stdio.h>
 
int main() {
 int a, b, c;
 scanf("%d", &c);
 scanf("%d %d", &a, &b);
 if(c % (a+b) == 0) printf("Farei hoje!\n");
 else printf("Deixa para amanha!\n");
    return 0;
}

File: C#/2717 Tempo do Duende.cs
using System;

class MainClass {
  public static void Main (string[] args) 
  {
    Console.Clear();
    int n = int.Parse(Console.ReadLine());
    string[] s = Console.ReadLine().Split(' ');
    int a = int.Parse(s[0]);
    int b = int.Parse(s[1]);
    if(n % (a+b) == 0) Console.WriteLine("Farei hoje!");
    else Console.WriteLine("Deixa para amanha!");
  }
}
